cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -173,46 +173,13 @@ func getNamespace() string {
 	return menv.Get(menv.OpenEBSNamespace)
 }
 
-// getStorageClass return storageclass object for a given storageClass Name.
-// or error if any.
-func getStorageClass(
-	scName string,
-) (*storagev1.StorageClass, error) {
-	if scName == "" {
-		return nil, errors.New("failed to get storageclass: name missing")
-	}
-	scObj, err := sc.NewKubeClient().Get(scName, metav1.GetOptions{})
-	if err != nil {
-		return nil, errors.Wrapf(
-			err,
-			"failed to get storageclass {%s}",
-			scName,
-		)
-	}
-	return scObj, nil
-}
-
-// getReplicationFactor gets the ReplicationFactor from the from given storageclass
-func getReplicationFactor(
-	class *storagev1.StorageClass,
-) (int, error) {
-
-	count := class.Parameters[ReplicaCount]
-
-	rfactor, err := strconv.Atoi(count)
-	if err != nil {
-		return 0, err
-	}
-	return rfactor, nil
-}
-
 // getSPC gets storagePoolClaim from
 // storageclass parameter
 func getSPC(
-	sc *storagev1.StorageClass,
+	claim *apis.CStorVolumeClaim,
 ) string {
 
-	spcName := sc.Parameters["storagePoolClaim"]
+	spcName := claim.Annotations["openebs.io/cspc-name"]
```

Comment:
annotation to label?

Reply:
changed to `claim.label["openebs.io/storage-pool-claim"]`